src/vl-map-search.js

```diff
@@ -1,108 +1,121 @@
+          }).then((data) => {
+            if (data && data.SuggestionResult) {
+              const resultaten = data.SuggestionResult.map((resultaat) => {
+                return {
+                  value: resultaat,
+                  label: resultaat
                 }
-            });
+              });
+              this._selectElement.choices = resultaten;
+            }
+          });
         }
+      });
     }
+  }
 
-    _addChoiceEventListener() {
-        if (!this.__choiceEventListenerRegistered) {
-            this.__choiceEventListenerRegistered = true;
-            this.__choiceEventListener = this._selectElement.addEventListener('choice', (event) => {
-                if (event && event.detail && event.detail.choice) {
-                    fetch(this.locationUrl + event.detail.choice.value)
-                        .then((response) => {
-                            return response.json();
-                        }).then((data) => {
-                            if (data && data.LocationResult) {
-                                this._map.zoomTo([data.LocationResult[0].BoundingBox.LowerLeft.X_Lambert72, data.LocationResult[0].BoundingBox.LowerLeft.Y_Lambert72, data.LocationResult[0].BoundingBox.UpperRight.X_Lambert72, data.LocationResult[0].BoundingBox.UpperRight.Y_Lambert72]);
-                            }
-                        });
+  _addChoiceEventListener() {
+    if (!this.__choiceEventListenerRegistered) {
+      this.__choiceEventListenerRegistered = true;
+      this.__choiceEventListener = this._selectElement.addEventListener(
+          'choice', (event) => {
+            if (event && event.detail && event.detail.choice) {
+              fetch(this.locationUrl + event.detail.choice.value)
+              .then((response) => {
+                return response.json();
+              }).then((data) => {
+                if (data && data.LocationResult) {
+                  if (this._onSelect) {
+                    this._onSelect(data);
+                  }
+                  this._map.zoomTo(
+                      [data.LocationResult[0].BoundingBox.LowerLeft.X_Lambert72,
+                        data.LocationResult[0].BoundingBox.LowerLeft.Y_Lambert72,
+                        data.LocationResult[0].BoundingBox.UpperRight.X_Lambert72,
+                        data.LocationResult[0].BoundingBox.UpperRight.Y_Lambert72]);
                 }
-            });
-        }
-    }
-
-    _configure() {
-        customElements.whenDefined('vl-map').then(() => {
-            if (this.parentNode  && this.parentNode.map && this.parentNode.host.tagName === 'VL-MAP') {
-                this.parentNode._shadow.prepend(this);
-                this.parentNode.host.style.setProperty('--vl-map--margin-top', "35px");
-                this._map = this._parentElement;
+              });
             }
-        });
+          });
     }
+  }
+
+  onSelect(callback) {
+    this._onSelect = callback;
+  }
```

Comment:
@ZhaodiFengVlaanderen ik heb deze logica verwijderd omdat het onderdeel is van #78 en #79. Bij het maken van een nieuwe branch best altijd van de master vertrekken zodat je niet ongewenst wijzigingen van andere branches overneemt.